lib/tasks/dev.rake

```diff
@@ -0,0 +1,43 @@
+namespace :dev do
+  require 'factory_bot_rails'
+  require 'faker'
+
+  desc 'Generate test data'
+  task :generate_data, [:dev_num] => :environment do |_, args|
+    set_args(args)
+
+    begin
+
+      res_pro_langs = create_programming_language
+      res_langs = create_language
+
+      @args[:dev_num].times do
+        email = loop do
+          email = "#{Faker::Twitter.screen_name[0..14]}@tech.reapra.sg" # The method can return a word with more than 15 chars.
+          break email unless Developer.exists?(email: email)
+        end
+
+        FactoryBot.create :developer, email: email,
+                          programming_languages: res_pro_langs.sample(rand(1..4)),
+                          languages: res_langs.sample(rand(1..3))
+
+        puts "Developer just created: #{email}"
+      end
+    rescue
+      puts "Data already exits, please check!"
```

Comment:
@ntrung2006 I think no need it at here, FactoryBot will do it for you
```
https://github.com/ntrung2006/exercise_find_good_develop/blob/feature_rake_task/spec/factories/developers.rb
```
change to
```
https://github.com/thienhv/example-ror/blob/master/spec/factories/developers.rb#L5
```

Reply:
Fixed!
Thank for your reviewed! @thienhv